resources/tiers.md

```diff
@@ -0,0 +1,84 @@
+# SDK tiers
+
+The objective of Meilisearch's Integrations Team is to **ease Meilisearch usage to as many developers as possible**.
+
+Given this statement, **tiers** are nothing more than a __group of SDKs/integrations__ where the integrations team can fairly divide their attention. Unfortunately is not possible to keep all the SDKs 100% updated against the new Meilisearch releases all the time, the Integrations Team is a polyglot small team that maintains 30+ different integrations and tools, so it is humanly impossible to consistently meet the Meilisearch quality standards.
+
+The ultimate goal of introducing tiers is to allow the team to have more time for new projects/integrations so that more time can be invested and more users will be impacted.
+
+It's been using [data collected here](https://docs.meilisearch.com/learn/what_is_meilisearch/telemetry.html) and data like GitHub stars, forks, and watchers to measure the impact of an SDK.
+
+It is worth remembering that this priority is just a concept introduced by the team where it aims to give a decision weight before
+we invest time in some issues of some SDK in question.
+
+:warning: We will keep all officially maintained SDKs **working**. :warning:
+
+## Tier #1
+
+The SDKs in group #1 have total priority over the team's time. Issues are visited before issues are present in tier #2, for example.
+It is worth remembering that this does not mean open issues will be finalized promptly. Every issue needs time to be understood/confirmed/worked on, and consequently closed.
+All new Meilisearch features that make sense for those SDKs will be available in the release day.
+
+## Tier #2
+
+Tier #2 SDKs are the emergent ones, we strongly believe they can move further, and we are betting on them.
+In any case, since they do not bring that much value compared to tier #1 we will only apply the amount of time required to keep it working and do important maintenance code and code cleanup.
```

Comment:
```suggestion
In any case, since they bring less value compared to tier #1 we only spend the amount of time required to keep it working and avoid important maintenance code and code cleanup.
```

What do you mean with `important maintenance` ?

Reply:
Actually, I'm referring to things the SDK should have, but they are not impacting all the users. It is harder to give a clear example, but I want to mean that we still can use some time on non-critical issues, you know?